addons/events/src/components/Event.tsx

```diff
@@ -5,10 +5,19 @@ import isEqual from 'lodash/isEqual';
 
 import { styled } from '@storybook/theming';
 import json from 'format-json';
-
 import Textarea from 'react-textarea-autosize';
+import { OnEmitEvent } from '../index';
+
+interface StyledTextareaProps {
+  shown: boolean;
+  failed: boolean;
+  value?: string;
```

Comment:
Probably I missed something... but putting only `...rest` didn't work. I also had to add `value`, `onChange` which seems unnecessary.

Reply:
You can also use something like `React.HTMLProps<HTMLTextAreaElement>`

Reply:
Thanks, I'll try that!

Reply:
hm... It didn't work for me. Maybe I implemented in a wrong way?